payment-service/src/main/java/com/hedvig/paymentservice/query/adyenAccount/AdyenAccount.kt

```diff
@@ -0,0 +1,13 @@
+package com.hedvig.paymentservice.query.adyenAccount
+
+import com.hedvig.paymentservice.domain.payments.enums.AdyenAccountStatus
+import javax.persistence.Entity
+import javax.persistence.Id
+
+@Entity
+data class AdyenAccount(
+    @Id
+    val memberId: String,
+    var recurringDetailReference: String,
+    var accountStatus: AdyenAccountStatus
+)
```

Comment:
A bonus that is usually good to have
```suggestion
@Entity
data class AdyenAccount(
    @Id
    val memberId: String,
    var recurringDetailReference: String,
    var accountStatus: AdyenAccountStatus
) {
   @field:CreationTimestamp(updatable=false)
   lateinit var createdAt: Instant
   @field:UpdateTimestamp
   lateinite var updatetAt: Instant
}
```